packages/devtools_app/lib/src/debugger/debugger_screen.dart

```diff
@@ -234,6 +240,51 @@ class DebuggerScreenBodyState extends State<DebuggerScreenBody>
       },
     );
   }
+
+  Widget _callStackRightChild() {
```

Comment:
instead of putting this in a method, just put CopyToClipboardControl directly in the `rightActions` list above. See my below comment - that will make the CopyToClipboardControl definition simpler